src/ontology/cl-edit.owl

```diff
@@ -30074,6 +30077,30 @@ AnnotationAssertion(rdfs:label obo:CL_4030008 "pronephric podocyte")
 EquivalentClasses(obo:CL_4030008 ObjectIntersectionOf(obo:CL_0000653 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002120)))
 SubClassOf(obo:CL_4030008 obo:CL_0000653)
 
+# Class: obo:CL_4030009 (epithelial cell of proximal tubule segment 1)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31249312"^^xsd:string) obo:IAO_0000115 obo:CL_4030009 "Epithelial cell that is part of segment 1 (S1) of the proximal tubule epithelium."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4030009 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030009 "2022-05-10T13:14:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4030009 "epithelial cell of proximal tubule segment 1"^^xsd:string)
+SubClassOf(obo:CL_4030009 obo:CL_0002306)
```

Comment:
@bvarner-ebi OLS is slow and I haven't had a chance at looking more closely in the ASCT+B table or recent Uberon commits: are new terms going to be created for proximal tubule segments, so that part-of axioms or equivalences can be added as appropriate?

Reply:
At this moment I was not planning to add the more specific Uberon terms. I agree they would be valuable, but I don't think they would be strictly necessary. My thought process is that we go this first pass with getting the new terms in CL so they will be available for use and we can always to a second pass of refining if desired. Does that seem like a reasonable approach? CC: @emquardokus

Reply:
@bvarner-ebi seems reasonable to me, yes.